flow/util/template_filters.py

```diff
@@ -172,6 +173,81 @@ def print_warning(msg):
     return ''
 
 
+def bold_font(s, eligible, pretty=False):
```

Comment:
I suggest we rename this filter-function to `strong()` and remove any logic from it. The filter should always return text that is presented "strongly" (bold). The logic on _when_ to use the filter should be in the template.